codecov.yml

```diff
@@ -0,0 +1,4 @@
+ignore:
+  - "src/Data.ts"
```

Comment:
Some troublesome to test files that codecov has mysteriously started complaining about...